### DISCUSSION TITLE
Be careful when you remove failed sectors

### DISCUSSION THREAD

**Discussion by de-authority** [filecoin-project/lotus#7293] [Storage Provider]
I found some of our storage providers get penalty after stopping seal sectors for one month, I believe this is related to MaxProveCommitDuration. If you complete precommit without complete provecommit, you will lose your precommit collateral. So be careful when you remove failed sectors.
 #### spec relate section
https://spec.filecoin.io/#section-systems.filecoin_mining.sector.adding_storage
#### Related code:
https://github.com/filecoin-project/specs-actors/blob/master/actors/builtin/miner/policy.go#L152
#### Related issue:
https://github.com/filecoin-project/specs-actors/issues/1433

**Comment by jennijuju** [comment]
Please post in non-announcement category.
